templates/html_report/composer.php

```diff
@@ -0,0 +1,100 @@
+<?php
+require __DIR__ . '/_header.php'; ?>
+
+<?php
+$packages = isset($project['composer']['packages']) ? $project['composer']['packages'] : [];
+usort($packages, function ($a, $b) {
+    return strcmp($a->name, $b->name);
+});
+if (empty($packages)) {
+    echo '<div class="row"><div class="column"><div class="bloc">No composer.json file found in this project</div></div></div>';
+    require __DIR__ . '/_footer.php';
+    return;
+}
+?>
+
+
+<div class="row">
+    <div class="column">
+        <div class="bloc">
+            <h4><?php echo sizeof($packages); ?> Composer dependencies</h4>
```

Comment:
`sizeof` is deprecated in favor of `count`

Reply:
Hi,

I don't think that [sizeof](https://www.php.net/sizeof) is deprecated. Do you have any info ?

Reply:
Sorry, my bad, I though it was deprecated in favor of count since it is an alias since long ago with no particular matter.
I found an RFC where sizeof was suggested as deprecated in PHP 7.2, but obviously, it was just a suggestion: https://wiki.php.net/rfc/deprecations_php_7_2#suggested_deprecations

Nevertheless, `sizeof` has different meanings accross all languages, while `count` is more oftenly considered as the native function used to count elements in an array, a set or a map. Also, I would prefer to use the expected function from PHP rather than an alias for it.

So, no, it is not deprecated, but I would suggest to prefer `count` :).

Reply:
No problem. I've updated code